uoapi/timetable/query_timetable.py

```diff
@@ -492,6 +497,21 @@ def normalize_whitespace(string):
 
 # Scraping
 
+def parse_available(s):
+    s = s.strip().lower()
+    assert len(s) == 4
+    s = s[1:]
+    try:
+        year = "20" + s[:2]
```

Comment:
could make it slightly fancier by getting the current year and stripping the last two digits, but that being said we don't have to worry about 2100 for quite some time

Reply:
Yeah, I think we can let the people dealing with this when we're dead handle it :p

Reply:
bold of you to assume I'll be dead by then